Release 2.0.0-rc4 (#2702)
timescale/timescaledb: CHANGELOG.md

`psql` with the `-X` flag to prevent any `.psqlrc` commands from
accidentally triggering the load of a previous DB version.**

## 2.0.0-rc4 (2020-12-02)

> This release candidate contains bugfixes since the previous release

Reviewer: I would expand (a bit dry right now). Maybe say something about improvements to COPY, support for gapfill, and improvements to chunk compression on distributed hypertables?

Author: Rewrote it. I'll add info about COPY when it is merged.

`psql` with the `-X` flag to prevent any `.psqlrc` commands from
accidentally triggering the load of a previous DB version.**

## 2.0.0-rc4 (2020-12-02)

This release candidate contains bugfixes since the previous release

> candidate, as well as additional minor features. It adds checks of

Reviewer: unclear what "checks of configuration" is or refers to. Maybe "It improves configuration validation for background jobs, ..."

Author: Reformulated.

configuration, adds support for gapfill on distributed tables, and
improvements to compression for distributed hypertables.

**Minor Features**
* #2689 Check configuration in alter_job and add_job
* #2696 Support gapfill on distributed hypertable
* #2468 Show more information in get_git_commit
* #2678 Include user actions into job stats view
* #2664 Fix support for complex aggregate expression
* #2672 Add hypertable to continuous aggregates view

**Bugfixes**
* #2688 Fix crash for concurrent drop and compress chunk
* #2666 Fix timeout handling in async library
* #2683 Fix crash in add_job when given NULL interval

**Thanks**

> 

Reviewer: Should add acknowledgements.

Author: Added two found in associated issues.

## 2.0.0-rc4 (2020-12-02)

This release candidate contains bugfixes since the previous release
candidate, as well as additional minor features. It improves
validation of configuration changes for background jobs, adds support
for gapfill on distributed tables, contains improvements to the memory 
handling for large COPY, and contains improvements to compression for
distributed hypertables.

**Minor Features**
* #2689 Check configuration in alter_job and add_job
* #2696 Support gapfill on distributed hypertable
* #2468 Show more information in get_git_commit
* #2678 Include user actions into job stats view
* #2664 Fix support for complex aggregate expression
* #2672 Add hypertable to continuous aggregates view
* #2662 Save compression metadata settings on access node
* #2707 Introduce additional db for data node bootstrapping

**Bugfixes**
* #2688 Fix crash for concurrent drop and compress chunk
* #2666 Fix timeout handling in async library
* #2683 Fix crash in add_job when given NULL interval
* #2698 Improve memory handling for remote COPY
* #2555 Set metadata for chunks compressed before 2.0

**Thanks**
* @francesco11112 for reporting memory issue on COPY

> * @Netskeh for reporting bug on time_bucket problem in continuous
>   aggregates

Reviewer 2: There was no problem with `time_bucket` itself but with using it in an expression.
```suggestion
* @Netskeh for reporting a bug with expressions over time_bucket in continuous
   aggregates
```
